Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./Footer.css";
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <a href="#"><i class="far fa-comment" /></a>
```

Comment:
Use `className`, not `class`!
Also, consider formatting multiple elements on separate lines:
```javascript
<a href="#">
  <i className="far fa-comment" />
</a> 
```